code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -80,8 +80,11 @@
 		msg += "[t_He] [t_is] wearing [wear_neck.get_examine_string(user)] around [t_his] neck.\n"
 
 	//eyes
-	if(glasses && !(SLOT_GLASSES in obscured))
-		msg += "[t_He] [t_has] [glasses.get_examine_string(user)] covering [t_his] eyes.\n"
+	if(!(SLOT_GLASSES in obscured))
+		if(glasses)
+			msg += "[t_He] [t_has] [glasses.get_examine_string(user)] covering [t_his] eyes.\n"
+		else if(eye_color == BLOODCULT_EYE && iscultist(src))

```

Comment:
This is kinda leaky if we allow pure red as eye color

Reply:
That's what the examine text is for, no? It makes sure there's no doubt that they're a cultist.

Reply:
Pure red is allowed, so people who become cultists with naturally pure red eyes will have this desc.

Reply:
I understand the issue. It seems pretty unlikely to happen unless you were explicitly trying to make it happen. I'm not sure how to fix this. I suppose I could check to see if the cult was already risen, or maybe it's possible to make BLOODCULT_EYE something that isn't selectable as an eye color, but the more issues with this PR the more I feel like I'm in over my head with code that I only partially understand.

Reply:
You can just add simple mob trait for this.

Reply:
Yeeeah you underestimate how many people have pure red eyes(the answer is too many)

Reply:
I think my sybil character does actually

Reply:
Up to you, might be nice for admin abuse if it doesn't check specific color.